src/AddTodoForm.js

```diff
@@ -1,21 +1,31 @@
 import React from "react";
 
+const AddTodoForm = ({onAddTodo}) => {
 
-const AddTodoForm = (props) => {
+    let [todoTitle, setTodoTitle] = React.useState();
```

Comment:
nit: prefer const whenever possible, this is the style guide I usually refer to https://google.github.io/styleguide/jsguide.html#features-use-const-and-let no need to read it, just wanted to give you a reference